NachoSpigot-API/pom.xml

```diff
@@ -88,6 +88,36 @@
             <artifactId>annotations</artifactId>
             <version>23.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.checkerframework</groupId>
+            <artifactId>checker-qual</artifactId>
+            <version>3.21.0</version>
+        </dependency>
+        <!-- Kyori api -->
```

Comment:
Why import the entire Kyori Adventure API if you're only going to use the `Key` class of it? Just download and import the class itself, don't import the entire API.

Reply:
It was changed.